### PR TITLE
refactor: move highlighted-element to cuc

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

* remove `@ovh-ux/translate-async-loader`
* add `@ovh-ux/ng-translate-async-loader`

ref: OM-318

34d21e98 — refactor: move highlighted-element to cuc

### Related PR

- [x] https://github.com/ovh-ux/manager/pull/217

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

File: client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js [hunk 1]
```diff
@@ -108,9 +108,9 @@
 
       this.InfrastructureService.setPreferredView('diagram');
 
-      // Hide highlighted-element on change state
+      // Hide cuc-highlighted-elementon change state
```
Comment (marie-j) [inline client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js]:
```suggestion
      // Hide cuc-highlighted-element on change state
```

File: client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js [hunk 2]
```diff
@@ -194,8 +194,8 @@
                   action: 'attach',
                 });
 
-                this.$rootScope.$broadcast('highlighed-element.show', `compute,${connectedIp.id},${connectedVmId}`);
-                this.model.currentLinkEdit.connection.addClass('highlighed-element highlighed-element-active');
+                this.$rootScope.$broadcast('cuc-highlighted-element.show', `compute,${connectedIp.id},${connectedVmId}`);
+                this.model.currentLinkEdit.connection.addClass('cuc-highlighted-elementcuc-highlighted-element-active');
```
Comment (marie-j) [inline client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js]:
```suggestion
                this.model.currentLinkEdit.connection.addClass('cuc-highlighted-element cuc-highlighted-element-active');
```

File: client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js [hunk 3]
```diff
@@ -991,10 +991,10 @@
               if (this.model.currentLinkEdit.connectionCurrent) {
                 // this.model.currentLinkEdit.connectionCurrent
                 //   .setHoverPaintStyle({ lineWidth : 8 });
-                this.model.currentLinkEdit.connectionCurrent.removeClass('highlighed-element highlighed-element-active');
+                this.model.currentLinkEdit.connectionCurrent.removeClass('cuc-highlighted-elementcuc-highlighted-element-active');
```
Comment (marie-j) [inline client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js]:
```suggestion
                this.model.currentLinkEdit.connectionCurrent.removeClass('cuc-highlighted-element cuc-highlighted-element-active');
```

File: client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js [hunk 4]
```diff
@@ -1032,10 +1032,10 @@
                 // If there are a connection already, highlight it
                 if (this.model.currentLinkEdit.connectionCurrent) {
                   this.model.currentLinkEdit.connectionCurrent.setHoverPaintStyle({ lineWidth: 4 });
-                  this.model.currentLinkEdit.connectionCurrent.addClass('highlighed-element highlighed-element-active');
+                  this.model.currentLinkEdit.connectionCurrent.addClass('cuc-highlighted-elementcuc-highlighted-element-active');
```
Comment (FredericEspiau) [inline client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js]: Space

File: client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js [hunk 5]
```diff
@@ -1062,7 +1062,7 @@
 
               // set connection style
               this.model.currentLinkEdit.connection.setPaintStyle({ strokeStyle: this.constructor.getLinkColor(this.model.currentLinkEdit.connectedIp.type), lineWidth: 8, dashstyle: '2 1' });
-              this.model.currentLinkEdit.connection.addClass('highlighed-element highlighed-element-active');
+              this.model.currentLinkEdit.connection.addClass('cuc-highlighted-elementcuc-highlighted-element-active');
```
Comment (FredericEspiau) [inline client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js]: Space

File: client/app/cloud/project/compute/cloud-project-compute.html [hunk 6]
```diff
@@ -17,4 +17,4 @@
 
 </div>
 
-<div class="highlighed-element-overlay animated-300 fadeIn" data-highlighed-element="compute"></div>
+<div class="cuc-highlighted-element-overlay animated-300 fadeIn" data-cuc-highlighted-element="compute"></div>
```
Comment (FredericEspiau) [inline client/app/cloud/project/compute/cloud-project-compute.html]:
```suggestion
<div class="cuc-highlighed-element-overlay animated-300 fadeIn" data-cuc-highlighed-element="compute"></div>
```
😄

Comment (cbourgois) [reply client/app/cloud/project/compute/cloud-project-compute.html]: 😠

**Pull request merged**